From 7e05cac1ebd7184d99b989ca8bcb3e47ce609291 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Krause?= <joerg.krause@embedded.rocks>
Date: Sun, 1 Feb 2015 21:53:47 +0100
Subject: [PATCH] tools only in no dot config targets
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Revert upstream commit c7ad5cbb1ef2c33883f0fa7d0455095004fd306d:

    Makefile: drop "tools-only" from no-dot-config-targets

This indirectly pulls in a build-dependency on libssl-dev used for FIT
image support, and possibly GPL/OpenSSL licensing incompatibility
issues.

Based on Debian patch from u-boot-tools:
http://ftp.de.debian.org/debian/pool/main/u/u-boot/u-boot_2014.10+dfsg1-2.1.debian.tar.xz

Signed-off-by: Jörg Krause <joerg.krause@embedded.rocks>
[fabio: adapt it to 2016.09]
Signed-off-by: Fabio Estevam <fabio.estevam@nxp.com>
[Fabrice: adapt it to 2021.04]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 50077027ba7..4aa5e7f8322 100644
--- a/Makefile
+++ b/Makefile
@@ -521,7 +521,7 @@ env_h := include/generated/environment.h
 
 no-dot-config-targets := clean clobber mrproper distclean \
 			 help %docs check% coccicheck \
-			 ubootversion backup tests check qcheck tcheck pylint \
+			 ubootversion backup tests tools-only check qcheck tcheck pylint \
 			 pylint_err
 
 config-targets := 0
-- 
2.25.1

